runtimes/aleph-alpine-3.13-python/init1.py

```diff
@@ -339,8 +349,11 @@ async def run_executable_http(scope: dict) -> Tuple[Dict, Dict, str, Optional[by
                 tries += 1
                 headers, body = await make_request(session, scope)
             except aiohttp.ClientConnectorError:
-                if tries > 20:
+                if errorHandler == True:
                     raise
+                if tries > 20:
```

Comment:
Would it make more sense to measure the time passed instead of counting attempts ?

Reply:
I think that count retries is enough and has sense, but yes, we can change it over time.